torch/csrc/LinalgBackend.cpp

```diff
@@ -0,0 +1,81 @@
+#include <torch/csrc/LinalgBackend.h>
```

Comment:
@ngimel Suggests that creating a new namespace from C++ like this shouldn't be necessary

Reply:
(@ngimel suspects you can just put the necessary functions/options into Python and call whatever C++ you like)

Reply:
If we want an enum type in python, those py/c++ bindings are necessary. Cudnn flags, cublas.matmul flags or other similar flags all use primitive type -- boolean.

I'm basically following the implementation of `torch.memory_format` (`at::MemoryFormat`), which has an enum type in python as well.

Reply:
As mentioned below, we now have pybind. So you can do significantly simpler things than what we had to do for memory format: https://github.com/pytorch/pytorch/blob/dbbb02474b9ad8044d061caee9fb0916da93101b/torch/csrc/Module.cpp#L970-L990